src/main/app/common/InterestData.ts

```diff
@@ -0,0 +1,15 @@
+import { Moment } from 'moment'
+
+export class InterestData {
```

Comment:
What about making this an interface? It does not really have to be a class at this point.

Reply:
done